custom/icds_reports/models/helper.py

```diff
@@ -3,6 +3,12 @@
 
 from django.db import models
 
+from corehq.blobs import get_blob_db
+
+ICDS_BUCKET = 'icds_blobdb'
+EXPIRED = 60 * 60 * 24
```

Comment:
These can expire in a week, so that we'll always have some extra copies if the task fails to run for some reason